src/Resources/js/modules/participation-update-handler.ts

```diff
@@ -191,4 +189,9 @@ export class ParticipationUpdateHandler {
         this.updateCheckBoxWrapper($checkbox);
         this.toggleTooltip($checkbox, TooltipLabel.OFFERED_MEAL);
     }
+
+    public static changeToAssignedSlot($checkbox: JQuery, slot: string){
+        let $assignedSlot = $checkbox.closest('.meal').find('[value='+ slot +']');
+        $assignedSlot.prop('selected', 'selected');
```

Comment:
you can write it in one line
$checkbox.closest('.meal').find('[value='+ slot +']').prop('selected', 'selected');

the variable is not required